.changeset/fresh-moles-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-tools/test-harness': major
```

Comment:
Do we want to major release now, or just a minor pre-release?

Reply:
i'll update to a minor pre-release